Source/WebCore/loader/CookieJar.cpp

```diff
@@ -132,15 +132,29 @@ void CookieJar::setCookies(Document& document, const URL& url, const String& coo
         ASSERT_NOT_REACHED();
 }
 
-bool CookieJar::cookiesEnabled(const Document&) const
+bool CookieJar::cookiesEnabled(const Document& document) const
```

Comment:
Are the changes to this function really needed? Isn't this only used by WebKitLegacy?

Reply:
Yes, this is only used for WebKitLegacy. I made the change anyways since the code is still there, and result is supposed to be consistent.